lib/errors.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
```

Comment:
tbf I tend to agree with some of this but right now we're not going against any style guides. Perhaps some of this warrants futher discussion. How about we do it at the regular Thursday gathering?